client/src/components/Submissions/Submissions.jsx

```diff
@@ -132,7 +132,7 @@ const Submissions = () => {
 
   const useSubmitterRole = () => {
     const query = useQuery({
-      queryKey: ['submitter-role'],
+      queryKey: 'submitter-role',
```

Comment:
Is this a relevant change?

Reply:
This was to undo the change that had happened most likely when I pointed this branch to the Compress branch instead of the Main branch. I think we had briefly talked about it yesterday.